Makefile

```diff
@@ -68,8 +68,8 @@ build-linux:
 	mkdir -p ./build
 	docker build --tag shentu ./
 	docker create --name temp shentu:latest
-	docker cp temp:/usr/local/bin/certikd ./build/
-	docker cp temp:/usr/local/bin/certikcli ./build/
+	docker cp temp:/shentu/build/certikd ./build/
+	docker cp temp:/shentu/build/certikcli ./build/
```

Comment:
according to local test, previous path was wrong and nothing can be copied out from there.